lib/login/accountCreation.dart

```diff
@@ -0,0 +1,216 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+
+import 'package:http/http.dart' as http;
+import 'package:off_top_mobile/login/login.dart';
+
+int repoNumber;
+
+class SignUp extends StatefulWidget {
+  const SignUp({@required this.email});
+  final String email;
+  @override
+  _SignUpState createState() => _SignUpState();
+}
+
+class _SignUpState extends State<SignUp> {
+  // Getting value from TextField widget.
+  //final idController = TextEditingController();
+  final TextEditingController firstNameController = TextEditingController();
+  final TextEditingController lastNameController = TextEditingController();
+  final TextEditingController cityController = TextEditingController();
+  final TextEditingController ageController = TextEditingController();
+  final TextEditingController genderController = TextEditingController();
+  final TextEditingController professionalController = TextEditingController();
+  final TextEditingController usernameController = TextEditingController();
+
+  bool visible = false;
+  Future<void> makePostRequest() async {
+    setState(() {
+      visible = true;
+    });
+    debugPrint('In postrequest');
```

Comment:
@frlzjosh Should we keep debugPrint?

Reply:
Yeah maybe we change the content in this statement to something like `Inserting new user: {....}`

Reply:
deugPrint() was for me to know where the code was jumping to. I'll change it to where it lets the developer/tester know the request was made and data was submitted.